11/hmw-11-02.py

```diff
@@ -0,0 +1,11 @@
+# Написати кастомний Exception клас, MyCustomException,
+# який має повідомляти "Custom exception is occured"
+
+class MyCustomException(Exception):
+    print('Custom exception is occurred')
```

Comment:
If you want to add a custom message, you can do it:
1. By passing the message to the call of your exception: raise MyCustomException('Custom exception is occured')
2. By setting this value as a default in the `__init__` method:
```
class MyCustomException(Exception):
    def __init__(self, message='Custom exception is occurred'):
        super().__init__(message)